src/Auth_Command.php

```diff
@@ -717,18 +886,38 @@ public function list( $args, $assoc_args ) {
 				if ( empty( $auths ) ) {
 					EE::warning( sprintf( 'Auth does not exists on %s', $site_url ) );
```

Comment:
Change it to error for consistency with rest of the codebase i.e. site list etc...

Reply:
Resolved.